Build metadata rework (#82)
konflux-ci/docs: docs/modules/ROOT/pages/how-tos/creating.adoc

spec:
  url: "https://github.com/konflux-ci/anothertestrepo"
--

== Finding the built images

> After a pipeline completes with a built artifact, you may want to test the resulting image to ensure that it works properly. The `IMAGE_URL` Tekton result will be set to the pullspec for the image. This can be discovered using the UI or the CLI.

Author: This is the `IMAGE_URL`. I didn't want to repeat the same beginning of the sentence twice. I can combine the sentences though.

> After a pipeline completes with a built artifact, you may want to test the resulting image to ensure that it works properly. The `IMAGE_URL` Tekton result will be set to the pullspec for the image. This can be discovered using the UI or the CLI.

Author: How about 
```suggestion
After a pipeline completes with a built artifact, you may want to test the resulting image to ensure that it works properly. The `IMAGE_URL` Tekton result (discoverable from the UI or CLI) should be set to the pullspec for the image.
```

+

> NOTE: Images build from pull request pipelines will be automatically deleted 5 days after they are built. This parameter can be changed by xref:/how-tos/configuring/customizing-the-build.adoc[customizing the build].

Author: ```suggestion
NOTE: {ProductName} automatically deletes images built for PR pipelines five days after building them.
```
Sounds good to me!

=== With the UI

All build PipelineRuns are visible in the {ProductName} UI. The location of these images in the OCI registry is reported on the *Activity* page.

.Procedure

In the console, complete the following steps to find the image pullspec for a completed PipelineRun:

. Navigate to the *Activity* > *Pipeline runs* tab.

. For the component whose SBOM you want to view, select its most recent pipeline run.

> . Find the *Results* section at the bottom of the page and look for the `IMAGE_URL` row. It should resemble `quay.io/redhat-user-workloads/workspace-tenant/application/component:tag`. The `IMAGE_DIGEST` provided can be used as an alternative mechanism for referencing the image. You will 

Author: ```suggestion
. Find the *Results* section at the bottom of the page and look for the `IMAGE_URL` row. It should resemble `quay.io/redhat-user-workloads/workspace-tenant/application/component:tag`. You can use the `IMAGE_DIGEST` provided as an alternate mechanism for referencing the image.
```
Thanks for catching the dangle. My mind was in circles trying to find the best restructure for the documentaiton.

=== With the CLI

After the build PipelineRuns are completed from git push events, the Components are updated with the location of the artifact in the OCI registry.

> The CLI

Author: It certainly does.
```suggestion
```
